### PR TITLE
chore: upgrade dependencies

### PR DESCRIPTION
**Pull request by valdo404**
* Upgrade scalapb
* Upgrade circe

### CODE REVIEW

**Review by StephaneBour — APPROVED** [review]

**Review by mikaelBrosset — APPROVED** [review]

**Review by rlebran — APPROVED** [review]

File: openrtb-json/src/test/scala/com/powerspace/openrtb/json/fixtures/BidRequestFixtures.scala [hunk 1]
```diff
@@ -75,7 +75,7 @@ object BidRequestFixtures {
     else RequestOneof.Request("native-string")
     BidRequest(
       id = "fmySKZNcTFcTPOurFYivufGxMtuSYpen",
-      at = Some(AuctionType.fromValue(value = 2)),
+      at = Some(AuctionType.fromValue( 2)),
```
Comment (StephaneBour) [inline openrtb-json/src/test/scala/com/powerspace/openrtb/json/fixtures/BidRequestFixtures.scala]: espace

Comment (valdo404) [reply openrtb-json/src/test/scala/com/powerspace/openrtb/json/fixtures/BidRequestFixtures.scala]: Bien vu !

Comment (valdo404) [reply openrtb-json/src/test/scala/com/powerspace/openrtb/json/fixtures/BidRequestFixtures.scala]: done

**Review by mikaelBrosset — APPROVED** [review]

**Review by sithis2a — APPROVED** [review]

**Review by emanuele-pirro — APPROVED** [review]

**Review by gontarzpawel — APPROVED** [review]
Revamp of scala-openrtb?! LGTM

**Comment by valdo404** [comment]
> Revamp of scala-openrtb?! LGTM

Man it's just an upgrade of dependencies. ;) but I think there is more to come.

**Review by malves — APPROVED** [review]

**Pull request merged**